keyboards/hub16/keymaps/default/keymap.c

```diff
@@ -0,0 +1,143 @@
+/* Copyright 2019 Josh Johnson
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Allow reading of encoder switch
+#define readInput(port, pin) (((*(&port - 2)) & (1 << pin)) ? 1 : 0)
+#define SWITCH_1 PF7
+#define SWITCH_1_PORT PORTF
+#define SWITCH_2 PD7
+#define SWITCH_2_PORT PORTD
+
+// Function key we are 'wrapping' usual key presses in
+#define KC_WRAP KC_F24
+
+// Tap Dance Declarations
+enum { TD_TO_LED = 0, TD_TO_DEFAULT = 1 };
+
+qk_tap_dance_action_t tap_dance_actions[] = {
+    // Tap once for HID, twice to toggle layer 1
+    [TD_TO_LED]     = ACTION_TAP_DANCE_DUAL_ROLE(_______, 1),
+    [TD_TO_DEFAULT] = ACTION_TAP_DANCE_DUAL_ROLE(_______, 0)};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT( /* Base */
+    KC_A, KC_B, KC_C, KC_D, \
+    KC_E, KC_F, KC_G, KC_H, \
+    KC_I, KC_J, KC_K, KC_L, \
+    KC_M, KC_N, KC_O, TD(TD_TO_LED) \
+  ),
+
+  [1] = LAYOUT( /* LED Control */
+    _______, RGB_MOD, RGB_RMOD, RGB_TOG, \
+    RGB_VAD, RGB_VAI, RGB_HUD, RGB_HUI, \
+    RGB_SAD, RGB_SAI, _______, _______, \
+    _______, _______, RESET, TD(TD_TO_DEFAULT) \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.